### ISSUE TITLE
Different colours?

### ISSUE THREAD

**Issue by ajtruckle** [EnlighterJS/Plugin.WordPress#234]
Hi
Is it just me or are these colours changing?

![Query1](https://user-images.githubusercontent.com/7985331/79324350-66896380-7f07-11ea-99d7-9be253cbffe0.png)

For example, `font-size` looks darker on line 203 and lighter on line 211.

Why would this happen?

**Comment by AndiDittrich** [comment]
use the element inspector of your webbrowser and check the color codes

**Comment by ajtruckle** [comment]
Good thinking!

- The first one is: `#085789` (class `kw1`)
- The second one is `#4284ae` (class `kw3`)

**Comment by AndiDittrich** [comment]
sry i didn't take a closer look....this is an issue related to the tokenizer engine in v2 - please use EnlighterJS v3 to fix it

**Comment by ajtruckle** [comment]
Ok, but EnlighterJS v3 is part of your beta WordPress plugin isn't it? You don't yet have copy I can use on my website with bbPress? Were there not some issues? Or is there another way to use EnlighterJS v3?

**Comment by AndiDittrich** [comment]
currently you can only fix your bbpress theme and remove the weak css rule

**Comment by ajtruckle** [comment]
> currently you can only fix your bbpress theme and remove the weak css rule

Sadly, that is beyond my knowledge. I don't really know what to fix. Minor things I am Ok to sniff out and tweak but bigger ...

**Comment by AndiDittrich** [comment]
i didn't have the time to fix it for you right now...i'm working on the v4 release

**Comment by ajtruckle** [comment]
I understand. I have just downloaded your latest beta and tried it on the staging site:

![Issue](https://user-images.githubusercontent.com/7985331/79326636-144a4180-7f0b-11ea-9d98-acea57d7a312.png)

I can see teh colour issue I raised is indeed resolved with EnlighterJS v3. But I don't remember how I can actually fix that problem with it is showing the raw code understand the formatted code in the bbPress topics.
